CHANGELOG.md

```diff
@@ -2,7 +2,15 @@
 
 All changes that impact users of this module are documented in this file, in the [Common Changelog](https://common-changelog.org) format with some additional specifications defined in the CONTRIBUTING file. This codebase adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## Unreleased [major]
+
+_Full changeset and discussions: [#34](https://github.com/OpenTermsArchive/deployment/pull/34)._
+
+> Development of this release was supported by the [French Ministry for Foreign Affairs](https://www.diplomatie.gouv.fr/fr/politique-etrangere-de-la-france/diplomatie-numerique/) through its ministerial [State Startups incubator](https://beta.gouv.fr/startups/open-terms-archive.html) under the aegis of the Ambassador for Digital Affairs.
+
+### Changed
```

Comment:
> I wouldn't say there is any "Changed" here… this is purely "Release 1.0.0" with zero functional changes, no? 🙂 So maybe just a notice, or otherwise:

```suggestion
### Added
```

Reply:
I understand… I feel neither is right because "change" or "added" actually refers to changes or additions in the API, and it has not changed nor seen any addition, and we are left to discuss if the _package_ itself has seen a change or an addition… I'd be fine with just the notice, really :) but nothing blocking here!

Reply:
I agree with you that notice would have been the best but I think the changelog parser will raise an error. I'm going to try locally.

Reply:
It seems to work! let's do that! 😉